content/hi/docs/setup/production-environment/turnkey-solutions.md

```diff
@@ -0,0 +1,12 @@
+---
+title:  टर्नकी क्लाउड सॉल्यूशंस
+content_type:  अवधारणा
```

Comment:
I think अवधारणा is accreditation. While in the [English version](https://github.com/kubernetes/website/blob/dev-1.22/content/en/docs/setup/production-environment/turnkey-solutions.md), the content_type is listed as concept. Could you please confirm @anubha-v-ardhan if this content_type was approved anywhere?

Reply:
Nope the content_type has to be concept only. I have not approved this earlier anywhere.